.storybook/preview.js

```diff
@@ -49,7 +49,7 @@ addParameters({
             order: [
                 'Introduction',
                 'Foundation',
-                ['Colors', 'Typography', 'Layout', 'Miscellaneous', 'Customisation'],
+                ['Colors', 'Typography', 'Layout', 'Miscellaneous', 'Customization'],
```

Comment:
While both customise and customize both are valid. We need to have only one being used majorly across the documentation. There might be other words like analyse, analyze. Changing only for customise might be pretty bad in that case.